Sources/AppCoinsSDK/Models/Services/Gamification/AppCoinGamificationService.swift

```diff
@@ -9,6 +9,6 @@ import Foundation
 
 internal protocol AppCoinGamificationService {
     
-    func getTransactionBonus(address: String, package_name: String, amount: String, currency: Coin, result: @escaping (Result<TransactionBonusRaw, TransactionError>) -> Void)
+    func getTransactionBonus(address: String, package_name: String, amount: String, currency: String, result: @escaping (Result<TransactionBonusRaw, TransactionError>) -> Void)
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done